package.json

```diff
@@ -1,11 +1,18 @@
 {
+  "name": "dataproofer",
```

Comment:
interesting! silly question, won't this cause a name conflict with the other package.json named "dataproofer"? https://github.com/dataproofer/Dataproofer/blob/dev/src/package.json#L2

Reply:
oh also seeing all the tests are failing now hm

Reply:
I think it's likely this is that's what is breaking it, maybe it should be `dataproofer-cli`